### ISSUE TITLE
Devel-ebug: t/pad.t, t/stack.t fail against perl-5.31.9

### ISSUE THREAD

**Issue by jkeenan** [plicease/Devel-ebug#19]
When tested today against perl-5.31.9 on FreeBSD-11, threaded, as part of CPAN-River-3000 testing, this module suffered test failures for the first time in the 5.31 development cycle.
```
[Devel-ebug-0.59] $ thisprove -vb t/pad.t t/stack.t
t/pad.t .... 
1..22
ok 1 - 3 has 
ok 2 - 4 has $q=1
ok 3 - 5 has $q=1,$w=2
ok 4 - 12 has $e=undef,$q=1,$w=2
ok 5 - 13 has $e=undef,$q=1,$w=2,$x=2,$z=1
ok 6 - 14 has $c=3,$e=undef,$q=1,$w=2,$x=2,$z=1
ok 7 - 6 has $e=3,$q=1,$w=2
ok 8 - 7 has $e=4,$q=1,$w=2
ok 9 - 9 has $e=5,$q=1,$w=2
ok 10
ok 11
ok 12
ok 13
ok 14
ok 15
ok 16
ok 17
ok 18
ok 19
ok 20
ok 21
not ok 22

#   Failed test at t/pad.t line 88.
#          got: '{...}'
#     expected: '$koremutake'
# Looks like you failed 1 test of 22.
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/22 subtests 
t/stack.t .. 
1..29
ok 1
ok 2
Can't call method "package" on unblessed reference at t/stack.t line 25, <GEN1> line 7.
# Looks like your test exited with -1 just after 2.
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 27/29 subtests 

Test Summary Report
-------------------
t/pad.t  (Wstat: 256 Tests: 22 Failed: 1)
  Failed test:  22
  Non-zero exit status: 1
t/stack.t (Wstat: 65280 Tests: 2 Failed: 0)
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 29 tests but ran 2.
Files=2, Tests=24,  4 wallclock secs ( 0.02 usr  0.02 sys +  0.67 cusr  0.29 csys =  1.00 CPU)
Result: FAIL
```
As this testing process installs the latest versions of all CPAN distros which are prerequisites for this one, you may wish to explore the impact of the latest version of the YAML distro on this one.

Thank you very much.
Jim Keenan

**Comment by eserte** [comment]
Statistical analysis shows that there are no pass reports if YAML 1.30 is installed (theta=0):
```
****************************************************************
Regression 'mod:YAML'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	     -0.0000	      0.3353	  -0.00
[1='eq_1.14']  	      1.0000	      0.4741	   2.11
[2='eq_1.15']  	      0.9000	      0.3516	   2.56
[3='eq_1.17']  	      0.7500	      0.3748	   2.00
[4='eq_1.20']  	      0.9375	      0.3456	   2.71
[5='eq_1.21']  	      0.7143	      0.3584	   1.99
[6='eq_1.22']  	      0.8333	      0.3490	   2.39
[7='eq_1.23']  	      0.8537	      0.3366	   2.54
[8='eq_1.24']  	      0.8039	      0.3385	   2.37
[9='eq_1.26']  	      0.8500	      0.3435	   2.47
[10='eq_1.27'] 	      0.9667	      0.3408	   2.84
[11='eq_1.28'] 	      0.5000	      0.3556	   1.41
[12='eq_1.29'] 	      0.8730	      0.3379	   2.58
[13='eq_1.30'] 	      0.0000	      0.3396	   0.00

R^2= 0.396, N= 384, K= 14
****************************************************************
```

**Comment by plicease** [comment]
Seems there is a breaking change in YAML 1.30 which is likely the cause
```
1.30 Mon 27 Jan 2020 11:09:46 PM CET
 - Breaking Change: Set $YAML::LoadBlessed default to false to make it more
   secure
```

**Comment by plicease** [comment]
Fixed with #22

**Comment by eserte** [comment]
Confirmed --- the latest from github passes with perl-5.31.11.
